tests/test_CEDNWBConverter.py

```diff
@@ -0,0 +1,67 @@
+from mease_lab_to_nwb import CEDNWBConverter
+import pynwb
+from pathlib import Path
+import pytest
+import os
+
+
+def test_cednwbconverter(tmp_path):
+    # read smrx file
+    file_recording = str((Path(__file__).parent / "data" / "TTLtest.smrx").resolve())
```

Comment:
It looks like you're used to using pretty sophisticated workflows with cached info - if interested, you can actually setup the CI to handle some test CED data as well; see [this workflow](https://github.com/catalystneuro/nwb-conversion-tools/blob/master/.github/workflows/update-testing-data.yml) we use for downloading/caching a test file from the test dataset to use within the [CI tests](https://github.com/catalystneuro/nwb-conversion-tools/blob/master/tests/test_gin.py) that reference the data within that environment.

Reply:
@CodyCBakerPhD thanks! I'll try this out. Should I also remove the smrx test file I added here and open a PR to add it to https://gin.g-node.org/NeuralEnsemble/ephy_testing_data/src/master/spike2 instead?

Reply:
They would _love_ some `.smrx` files! That's currently a gap in both our coverage @samuelgarcia and @JuliaSprenger (the managers of that repo) would be happy to help get some in - the only thing I'd mention is it's critical for CI workflows that the example file that gets uploaded is very small (< 10 MB, ideally) while still containing all relevant header info + a snippet of data. That's *usually* comparable to a minute or less of recording, or simply an existing recording with all the data after that point clipped off.

Reply:
Ahh, I see it - wasn't looking under `spike2`! Thanks Sam